### PR TITLE
Adding limitations for 5.1

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/uV9FuKGO/85-5-1-limitations and https://trello.com/c/mTr7xcBp/250-5-1-known-limitations

### CODE REVIEW

**Comment by hflynn** [comment]
Still awaiting file format list from Melissa and me finding somewhere to put the ROI/rendering settings point for devs.

**Comment by hflynn** [comment]
Will need rebasing to dev_5_0

**Comment by hflynn** [comment]
Last commit NOT for rebasing - @jburel you'd closed the linked ticket http://trac.openmicroscopy.org.uk/ome/ticket/3256 with a comment that it is implemented in 5.1.0-m1 so I assume the format now works, please let me know if not!

**Comment by jburel** [comment]
@hflynn: we rebased all the work to 5.0.x so the comment can be removed from 5.0 doc too.
The problem was not format specific but pixel type specific, yes the format works.

**Comment by hflynn** [comment]
Excellent, thanks!

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/limitations.html.

File: omero/sysadmins/limitations.txt [hunk 1]
```diff
@@ -109,14 +118,16 @@ group that data is in into a private group.
...
 -------------------
 
-DeltaVision OMX files (.hdr) not viewable
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Large images with floating-point pixel data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pyramids of image tiles are currently not generated for images with
+floating-point pixel data, meaning the imported image will be scrambled if it
+is over a certain size. This primarily affects the following file formats:
```
Comment (mtbc) [inline omero/sysadmins/limitations.txt]: Can we say what size?

**Comment by mtbc** [comment]
Reads fine.

File: omero/sysadmins/limitations.txt (hunk 1 above)
Comment (hflynn) [inline omero/sysadmins/limitations.txt]: @melissalinkert is there a set size we can give here?

Comment (rleigh-dundee) [inline omero/sysadmins/limitations.txt]: I don't think so; it's configurable.  We could mention the default though (though I can't find where it lives offhand)

Comment (sbesson) [inline omero/sysadmins/limitations.txt]: Defaults are set by https://www.openmicroscopy.org/site/support/omero5.1/sysadmins/config.html#omero-pixeldata-max-plane-height and https://www.openmicroscopy.org/site/support/omero5.1/sysadmins/config.html#omero-pixeldata-max-plane-width

**Comment by rleigh-dundee** [comment]
Looks fine other than the size noted.  A cross-reference to the settings would fix that.

**Comment by hflynn** [comment]
Yep, just pushing a fix thanks!

**Comment by sbesson** [comment]
Independently of this PR, since we are introducing cross-links to these pyramid properties, it might be sensible to add a minimal description for both of them under `etc/omero.properties` which would expand the configuration page.

**Comment by rleigh-dundee** [comment]
@hflynn The correction looks fine, thanks.

**Comment by joshmoore** [comment]
> https://github.com/openmicroscopy/ome-documentation/pull/1138#issuecomment-78941110

I'm happy to do another round of adding to etc/omero.properties once my other PRs are in.

**Comment by hflynn** [comment]
--rebased-to #1141

**Pull request merged**